src/node_binding.cc

```diff
@@ -243,8 +243,7 @@ using v8::Value;
 // Globals per process
 static node_module* modlist_internal;
 static node_module* modlist_linked;
-static uv_once_t init_modpending_once = UV_ONCE_INIT;
-static uv_key_t thread_local_modpending;
+static thread_local struct node_module* thread_local_modpending = nullptr;
```

Comment:
Just `node_module*`, no struct (see lines above), and the `= nullptr` isn't strictly necessary.

Reply:
👍 on the `node_module*`, but I'd like to keep the initialization because we do a
```C++
CHECK_NULL(thread_local_modpending);
```
later on, and I'm not 100% sure that these variables get initialized on every platform.

Reply:
OK, looks like the variable oughta get zeroed out: https://en.cppreference.com/w/cpp/language/initialization#Static_initialization